lib/routes/devices-sessions.js

```diff
@@ -403,11 +403,12 @@ module.exports = (log, db, config, customs, push, devices) => {
         const id = request.payload.id
         let result
 
-        return push.notifyDeviceDisconnected(uid, id)
-          .catch(() => {})
-          .then(() => db.deleteDevice(uid, id))
+        return db.deleteDevice(uid, id)
           .then(res => {
             result = res
+          })
+          .then(() => push.notifyDeviceDisconnected(uid, id).catch(() => {}))
```

Comment:
Can you stick the `.catch` on a new line? I thought it had disappeared at first, didn't see it hiding over on the rhs.